scripts/privyr-generic-cf-integration.js

```diff
@@ -66,6 +70,17 @@ export default class PrivyrGenericCfIntegration {
         }
     };
 
+    _sending_beat() {
+        let payload = {
+            "license_code": this.license_code,
+            "full_url": window.location.href,
```

Comment:
:point_up:  We may not need the GET param right? For ex:
Same form on `www.privyr.com?gclid=232bfjhbf2` and `www.privyr.com?gclid=asafqw` should be same heartbeat. 

What do you think?